pkg/cloudprovider/providers/vsphere/vsphere_util_others.go

```diff
@@ -0,0 +1,53 @@
+// +build !windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vsphere
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+)
+
+const (
+	UUIDPath   = "/sys/class/dmi/id/product_serial"
```

Comment:
The sysfs path `/sys/..` is a bit Linux specific. How about naming this file `vsphere_util_linux.go` (rather than `vsphere_util_others.go`) and have a separate `vsphere_util_unsupported.go`?

Reply:
Thanks .. the new structure LGTM